src/crate/client/test_http.py

```diff
@@ -436,15 +436,30 @@ def test_remove_certs_for_non_https(self):
         self.assertTrue('foobar' in d)
 
 
-class RetryRequestHandler(BaseHTTPRequestHandler):
+class TimeoutRequestHandler(BaseHTTPRequestHandler):
     """
-    http handler for use with HTTPServer
+    HTTP handler for use with TestingHTTPServer
+    updates the shared counter and waits so that the client times out
+    """
+
+    def do_POST(self):
+        print('POST /_sql')
```

Comment:
Do we really want to have print statements here?

Reply:
No